torch/testing/_internal/common_quantization.py

```diff
@@ -307,6 +307,25 @@ def wrapper(*args, **kwargs):
             fn(*args, **kwargs)
     return wrapper
 
+def withQNNPACKBackend(fn):
```

Comment:
btw, should we allow an argument for this one as well so that it can be reused by other backends like fbgemm, onednn?

Reply:
yeah, that makes sense.  Maybe as a separate PR? Would be a good BE task to simplify some of our tests.

Reply:
sure, sounds good